pulp_ansible/app/viewsets.py

```diff
@@ -790,6 +887,47 @@ def sign(self, request, pk):
         )
         return OperationPostponedResponse(result, request)
 
+    @extend_schema(
+        description="Trigger an asynchronous task to sign Ansible content with Sigstore.",
+        responses={202: AsyncOperationResponseSerializer},
+    )
+    @action(
+        detail=True, methods=["post"], serializer_class=AnsibleRepositorySigstoreSignatureSerializer
+    )
+    def sigstore_sign(self, request, pk):
+        """
+        Dispatches a Sigstore signing task.
+        This endpoint is in tech preview and can change at any time in the future.
+        """
```

Comment:
Would it be possible to move this into the existing `sign` endpoint? From what I've seen it's not that different call argument wised to what we have already. If you turn the `SigstoreSigningService` to be of type `SigningService` then you can use similar logic like we have in `sync` and use the type of the signing service to determine which task to call.